kafka/testresources/kraft/server.properties

```diff
@@ -28,4 +26,4 @@ controller.listener.names=CONTROLLER
 listener.security.protocol.map=PLAINTEXT:PLAINTEXT,SSL:SSL,SASL_PLAINTEXT:SASL_PLAINTEXT,SASL_SSL:SASL_SSL,CONTROLLER:SASL_PLAINTEXT,DOCKER:PLAINTEXT,DOCKER_SASL_PLAINTEXT:SASL_PLAINTEXT
 authorizer.class.name=org.apache.kafka.metadata.authorizer.StandardAuthorizer
 sasl.enabled.mechanisms=PLAIN
-controller.quorum.voters=0@0.0.0.0:38705
\ No newline at end of file
+controller.quorum.voters=0@localhost:38705
```

Comment:
You can put the docker network node name: `kafka` that'll be useful when expanding cluster side